### PR TITLE
validate PDB creation post deployment

### PR DESCRIPTION
**Pull request by annagitel**
addressing  https://github.com/red-hat-storage/ocs-ci/issues/1215

Signed-off-by: asandler <asandler@localhost.localdomain>

### CODE REVIEW

File: ocs_ci/ocs/cluster.py [hunk 1]
```diff
@@ -552,3 +552,46 @@ def validate_cluster_on_pvc():
                 assert claimName in pvc_names, (
                     "Ceph Internal Volume not backed by PVC"
                 )
+
+
+def count_cluster_osd():
+    """
+    Returns the number of OSD pods in current cluster
+
```
Comment (RazTamir) [inline ocs_ci/ocs/cluster.py]: Please add return type

File: ocs_ci/ocs/cluster.py [hunk 2]
```diff
@@ -552,3 +552,46 @@ def validate_cluster_on_pvc():
...
+        pdb_required.append(constants.OSD_PDB + str(num))
+
+    pdb_list.sort()
+    pdb_required.sort()
+    logger.info(pdb_list)
+    logger.info(pdb_required)
+    for required, given in zip(pdb_required, pdb_list):
+        assert required == given, f"{required} was not created"
+
+    logger.info("All required PDBs created.")
```
Comment (ebenahar) [inline ocs_ci/ocs/cluster.py]: Please add which PDBs were checked:

```python
logger.info(f"The following required PDBs exist: {required}")
```

File: ocs_ci/ocs/cluster.py [hunk 3]
```diff
@@ -552,3 +552,45 @@ def validate_cluster_on_pvc():
                 assert claimName in pvc_names, (
                     "Ceph Internal Volume not backed by PVC"
                 )
+
+
+def count_cluster_osd():
+    """
+    Returns:
+         osc_count (Integer): number of OSD pods in current cluster
```
Comment (RazTamir) [inline ocs_ci/ocs/cluster.py]: Int instead of Integer

**Comment by annagitel** [comment]
Moving to new PR duo to Travis problem

**Pull request closed without merging**